magento-engcom/import-export-improvements
# magento-engcom/import-export-improvements#75: fix stoping on errors

Fix stoping on errors so that it uses isErrorLimitExceeded instead of count errors, and include checking critical errors in isErrorLimitExceeded.





### Description (*)


### Fixed Issues (if relevant)

1. magento-engcom/import-export-improvements#75

### Manual testing scenarios (*)

1. Set skip errors on import,
2. Set limit to 10
3. Import file with 9 errors,
4. Message should say file is valid but with 9 errors and the user should be able to import,

### Contribution checklist (*)
